front-end/src/components/common/SubscriptionsTable/index.jsx

```diff
@@ -38,12 +41,25 @@ const TeacherTable = ({ subscriptions, deleteSubscriptionById }) => {
   return (
     <TableContainer component={Paper} className={styles.container}>
       <Table sx={{ minWidth: 650 }} aria-label='simple table'>
-        <TableHead>
+        <TableHead
+          sx={{
+            '& th': {
+              color: 'white',
+              backgroundColor: '#a968a3',
+              fontSize: 16,
+              fontWeight: 500,
+            },
+          }}
+        >
           <TableRow>
-            <TableCell>Avatar</TableCell>
-            <TableCell align='left'>Full Name</TableCell>
-            <TableCell align='left'>Email</TableCell>
-            <TableCell align='left'>Following</TableCell>
+            <TableCell sx={{ maxWidth: 65 }} align='center'>
+              Avatar
+            </TableCell>
+            <TableCell align='center'>Full Name</TableCell>
```

Comment:
Better create const for tableCell and use method map

Reply:
fix it